src/HotChocolate/Core/test/Validation.Tests/__snapshots__/RequiredArgumentRuleTests.IgnoresUnknownArguments.snap

```diff
@@ -0,0 +1,22 @@
+﻿[
+  {
+    "Message": "The argument `unknownArgument` does not exist.",
```

Comment:
to what test does this snapshot belong to?

Reply:
Good catch.. this one is not needed

Reply:
Can we remove it or is it used?

Reply:
Done